CRM/Contact/BAO/GroupContactCache.php

```diff
@@ -796,7 +796,10 @@ private static function updateCacheFromTempTable(CRM_Utils_SQL_TempTable $groupC
    */
   protected static function insertGroupContactsIntoTempTable(string $tempTableName, int $groupID, ?int $savedSearchID, ?string $children): void {
     if ($savedSearchID) {
-      $ssParams = CRM_Contact_BAO_SavedSearch::getSearchParams($savedSearchID);
```

Comment:
I've basically copied getSearchParams into the appropriate parts of this function so we can start from a clean array